src/tribler-core/tribler_core/modules/metadata_store/restapi/tests/test_metadata_endpoint.py

```diff
@@ -203,18 +202,13 @@ async def setUp(self):
         self.udp_port = self.get_port()
         self.udp_tracker = UDPTracker(self.udp_port)
 
-        self.http_port = self.get_port()
-        self.http_tracker = HTTPTracker(self.http_port)
-
     async def tearDown(self):
         self.session.dlmgr = None
         if self.udp_tracker:
             await self.udp_tracker.stop()
-        if self.http_tracker:
-            await self.http_tracker.stop()
         await super(TestTorrentHealthEndpoint, self).tearDown()
 
-    @timeout(TORRENT_CHECK_TIMEOUT + 5)
+    @timeout(5)
```

Comment:
Why was this change made? 🤔

Reply:
This high timeout of 25 seconds was actually masking the problem: the `UDPTracker` was broken, causing the test to take 20 seconds. Because this was still within the allowed timeout, it was marked as successful.

Reply:
Ok, thanks! 👍